imageio/imageio-dds/src/test/java/com/twelvemonkeys/imageio/plugins/dds/DDSImageReaderTest.java

```diff
@@ -57,6 +57,7 @@ protected List<TestData> getTestData() {
         Dimension dim1 = new Dimension(1, 1);
 
         return Arrays.asList(
+                new TestData(getClassLoaderResource("/dds/dxt10.dds"), dim256),
```

Comment:
Excellent, thanks for providing a test case! I prefer, however, if you append it to the end of the list. 😀